docker/conf.d/auth.conf

```diff
@@ -0,0 +1,193 @@
+authorization: {
```

Comment:
I notice that this file doesn't have the `certificate_status` endpoints exposed. Is that intentional?

Reply:
no, i just copied these from wherever i thought the packages got them